src/tests/functional/shared_test_classes/include/shared_test_classes/base/benchmark.hpp

```diff
@@ -0,0 +1,181 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cctype>
+#include <iostream>
+
+#include "layer_test_utils.hpp"
+#include "ov_subgraph.hpp"
+
+namespace LayerTestsDefinitions {
+
+template <typename BaseLayerTest>
+class BenchmarkLayerTest : public BaseLayerTest {
+    static_assert(std::is_base_of<LayerTestsUtils::LayerTestsCommon, BaseLayerTest>::value,
+                  "BaseLayerTest should inherit from LayerTestsUtils::LayerTestsCommon");
+
+public:
+    void Run(const std::initializer_list<std::string>& names,
+             const std::chrono::milliseconds warmupTime = std::chrono::milliseconds(2000),
+             const int numAttempts = 100) {
+        bench_names_ = names;
+        warmup_time_ = warmupTime;
+        num_attempts_ = numAttempts;
+        LayerTestsUtils::LayerTestsCommon::configuration = {{"PERF_COUNT", "YES"}};
+        LayerTestsUtils::LayerTestsCommon::Run();
+    }
+
+    void Run(const std::string& name,
+             const std::chrono::milliseconds warmupTime = std::chrono::milliseconds(2000),
+             const int numAttempts = 100) {
+        if (!name.empty()) {
+            Run({name}, warmupTime, numAttempts);
+        } else {
+            Run({}, warmupTime, numAttempts);
+        }
+    }
+
+    void Validate() override {
+        // NOTE: Validation is ignored because we are interested in benchmarks results.
```

Comment:
MINOR: move comment to the above of method

Reply:
Done